tgmc.dme

```diff
@@ -1,1452 +1,1452 @@
-// DM Environment file for baystation12.dme.
```

Comment:
You might want to revert this file

Reply:
Oh, I'll take a look again tomorrow, I'm just on my way to bed. Thanks